src/runtime/api-channel.ts

```diff
@@ -483,16 +482,16 @@ export abstract class PECOuterPort extends APIPort {
   abstract onRender(particle: recipeParticle.Particle, slotName: string, content: Content);
   abstract onInitializeProxy(handle: UnifiedStore, callback: number);
   abstract onSynchronizeProxy(handle: UnifiedStore, callback: number);
-  abstract onHandleGet(handle: StorageProviderBase, callback: number);
-  abstract onHandleToList(handle: StorageProviderBase, callback: number);
-  abstract onHandleSet(handle: StorageProviderBase, data: {}, particleId: string, barrier: string);
-  abstract onHandleClear(handle: StorageProviderBase, particleId: string, barrier: string);
-  abstract onHandleStore(handle: StorageProviderBase, callback: number, data: {value: {}, keys: string[]}, particleId: string);
-  abstract onHandleRemove(handle: StorageProviderBase, callback: number, data: {}, particleId: string);
-  abstract onHandleRemoveMultiple(handle: StorageProviderBase, callback: number, data: {}, particleId: string);
-  abstract onHandleStream(handle: StorageProviderBase, callback: number, pageSize: number, forward: boolean);
-  abstract onStreamCursorNext(handle: StorageProviderBase, callback: number, cursorId: number);
-  abstract onStreamCursorClose(handle: StorageProviderBase, cursorId: number);
+  abstract onHandleGet(handle: UnifiedStore, callback: number);
+  abstract onHandleToList(handle: UnifiedStore, callback: number);
+  abstract onHandleSet(handle: UnifiedStore, data: {}, particleId: string, barrier: string);
+  abstract onHandleClear(handle: UnifiedStore, particleId: string, barrier: string);
+  abstract onHandleStore(handle: UnifiedStore, callback: number, data: {value: {}, keys: string[]}, particleId: string);
+  abstract onHandleRemove(handle: UnifiedStore, callback: number, data: {}, particleId: string);
+  abstract onHandleRemoveMultiple(handle: UnifiedStore, callback: number, data: {}, particleId: string);
+  abstract onHandleStream(handle: UnifiedStore, callback: number, pageSize: number, forward: boolean);
+  abstract onStreamCursorNext(handle: UnifiedStore, callback: number, cursorId: number);
+  abstract onStreamCursorClose(handle: UnifiedStore, cursorId: number);
```

Comment:
I think all of these are obsoleted by onProxyMessage.

Reply:
Thanks, done! Added comment too